src/Components/MovieContainer/MovieContainer.js

```diff
@@ -1,9 +1,9 @@
 import './MovieContainer.css'
 import MovieCard from '../MovieCard/MovieCard.js'
 
-const MovieContainer = ({movies, handleClick}) => {
+const MovieContainer = ({movies}) => {
   const moviesData = movies.map(movie => {
-    return (<MovieCard key={movie.id} handleClick={handleClick} movieInfo={movie}/>)
+    return (<MovieCard key={movie.id}  movieInfo={movie}/>)
```

Comment:
You could consider pulling this map out into its own variable (this is called memoizing), and then referencing that variable in your return. It keeps the return very clean and readable!

Reply:
Thats a very good idea, I'll implement that change